static/js/components/PersonalTab.js

```diff
@@ -57,13 +57,14 @@ class PersonalTab extends ProfileTab {
       )}<br />
       <h4>Where do you live?</h4>
       {this.boundTextField(['city'], 'City')}<br />
-      {this.boundTextField(['state_or_territory'],'State or Territory')}<br />
+      {this.boundStateSelectField(['state_or_territory'], ['country'], 'State or Territory')}<br />
```

Comment:
In my fantasy, the "State of Territory" component wasn't displayed at all until a country was chosen. But I was assuming that we couldn't get a complete set of states for all countries.